testflinger_agent/client.py

```diff
@@ -267,10 +292,42 @@ def post_agent_data(self, data):
         :param data:
             dict of various agent data points to send to the api server
         """
-        agent = self.config.get("agent_id")
         agent_data_uri = urljoin(self.server, "/v1/agents/data/")
-        agent_data_url = urljoin(agent_data_uri, agent)
+        agent_data_url = urljoin(agent_data_uri, self.config.get("agent_id"))
         try:
             self.session.post(agent_data_url, json=data, timeout=30)
         except RequestException as exc:
             logger.error(exc)
+
+    def post_influx(self, job_id, phase, duration, result):
+        """Post the relevant data points to testflinger server
+
+        :param data:
+            dict of various agent data points to send to the api server
+        """
+        measurement = "%s phase result" % job_id
+        data = [
+            {
+                "measurement": measurement,
+                "tags": {
+                    "agent": self.config.get("agent_id"),
+                },
+                "fields": {
+                    "phase": phase,
+                    "duration": duration,
+                    "result": result,
+                },
+                "time": int(time.time()),
```

Comment:
I don't think you actually need to include this. I'm pretty sure influxdb automatically adds timestamp.

Reply:
Documentation I've seen includes this, but might as well be reductive and see if its necessary.

Reply:
Looked into this a bit and it appears that time is a required field like measurement.